docs/howto/environment.md

```diff
@@ -15,35 +15,41 @@
   ```
   For comparison devs have used node at 10.13.x and 12.x.x.
 
-3. Install development copies of cylc and cylc-uiserver:
+3. Install
+   [cylc/cylc-flow](https://github.com/cylc/cylc-flow/) and
+   [cylc/cylc-uiserver](https://github.com/cylc/cylc-uiserver/):
   ```bash
-  pip install -e $DEVDIR/cylc
-  pip install -e $DEVDIR/cylc-uiserver
+  pip install -e "${DEVDIR}/cylc-flow" -e "${DEVDIR}/cylc-uiserver"
   ```
-4. Install npm in cylc-ui client.
+  where `${DEVDIR}/cylc-flow` and `${DEVDIR}/cylc-uiserver` are the locations
+  of your local (git) worktrees of the two projects.
+
+4. Install [cylc/cylc-ui](https://github.com/cylc/cylc-ui/).
   ```bash
   cd $DEVDIR/cylc-ui
-  npm install
+  npm install --prefix="${DEVDIR}/cylc-ui"
   npm install -g configurable-http-proxy
   ```
+  where `${DEVDIR}/cylc-ui` is the location
+  of your local (git) worktree of the project.
 
-5. in `$DEVDIR/cylc-uiserver/jupyterhub_config.py` change
+5. Edit `${DEVDIR}/cylc-uiserver/jupyterhub_config.py`. Change:
 ```diff
 -c.Spawner.args = ['-s', '../cylc-ui/dist/']
 +c.Spawner.args = ['-s', '/path/to/my/home/metomi/cylc-ui/dist/']
 ```
+where `/path/to/my/home/metomi/cylc-ui` is the value of `${DEVDIR}/cylc-ui`.
 
-6. Set `jupyterhub` running.
-
-7. Open a new terminal and ensure that you have the correct conda environment.
+6. Run `jupyterhub`.
   ```bash
   conda activate my-example-env
-  cd $DEVDIR/cylc-ui
+  jupyterhub -f ${DEVDIR}/cylc-uiserver/jupyterhub_config.py
   ```
 
-8. Run the UI
+7. Run UI.
   ```bash
-  npm run build:watch
+  conda activate my-example-env
+  npm run --prefix="${DEVDIR}/cylc-ui" build:watch
   ```
 
 * Open a browser and navigate to `localhost:8000`. You will be asked to log in
```

Comment:
Not really part of main req, but why not?
Suggest modify to 
[localhost:8000](https://localhost:8000)

Reply:
Probably `https` won't be enabled in development... though that's possible too. Maybe `http://localhost:8000`?

Reply:
Modified to an unwrapped `http://localhost:8000/` string.